index.bs

```diff
@@ -4532,39 +4735,31 @@ partial interface MLGraphBuilder {
 };
 </script>
 
+{{MLReduceOptions}} has the following members:
+<dl dfn-type=dict-member dfn-for=MLReduceOptions>
+    : <dfn>axes</dfn>
+    ::
+        A sequence of {{unsigned long}}. The dimensions to reduce. The values in the sequence must be in the range [0, N-1] where N is the [=rank=] of the input tensor. If not present, all dimensions are reduced.
+
+    : <dfn>keepDimensions</dfn>
+    ::
+        A {{boolean}}. If true, retains reduced dimensions with [=list/size=] 1. The default value is false.
```

Comment:
+1 to have standalone `argMax` and `argMin`.

For Chromium prototyping, adding `axis_direction` field into [`Reduce`](https://source.chromium.org/chromium/chromium/src/+/main:services/webnn/public/mojom/webnn_graph.mojom;l=255) mojo message is not ideal, because it would only apply to `argMax` and `argMin` enum values. This is not recommended by Chromium mojo guideline that requires ["All possible message values are semantically valid"](https://chromium.googlesource.com/chromium/src/+/main/docs/security/mojo.md#All-possible-message-values-are-semantically-valid).

Reply:
Can this be addressed in a separate issue as it pre-exists this PR? I'll separate argMin/argMax out of reduction functions.

Reply:
Fixed.